html/elements/meta.json

```
@@ -675,10 +675,12 @@
                   "version_added": false
                 },
                 "safari": {
-                  "version_added": false
+                  "version_added": "15",
+                  "notes": "Safari 15 beta supports a media attribute that recognizes (prefers-color-scheme: light) or (prefers-color-scheme: dark)."
                 },
                 "safari_ios": {
-                  "version_added": false
+                  "version_added": "15",
+                  "notes": "Safari 15 beta supports a media attribute that recognizes (prefers-color-scheme: light) or (prefers-color-scheme: dark) on iPadOS 15."
```

And a bonus one:

4. Where possible, front-load exceptions, so readers can tell if it's relevant to them without reading to the very end.

```suggestion
                  "notes": "On iPadOS, Safari supports a media attribute that recognizes <code>prefers-color-scheme: light</code> or <code>prefers-color-scheme: dark</code>."
```

---

Ditto above a new entry here.